DigitalLearningSolutions.Data/Helpers/NumberOfAdministratorsHelper.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace DigitalLearningSolutions.Data.Helpers
+{
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.DataServices.UserDataService;
+    using DigitalLearningSolutions.Data.Models;
+
+    public interface INumberOfAdministratorsHelper
+    {
+        NumberOfAdministrators GetCentreAdministratorNumbers(int centreId);
+
+    }
+
+    public class NumberOfAdministratorsHelper: INumberOfAdministratorsHelper

```

Comment:
Is there any particular reason this is a Helper not a Service? It seems more like a service to me, with helpers usually being static.